pkg/engine/source/source.go

```diff
@@ -29,3 +35,41 @@ type QueriesSource interface {
 	GetQueries(querySelection *QueryInspectorParameters) ([]model.QueryMetadata, error)
 	GetQueryLibrary(platform string) (string, error)
 }
+
+// mergeLibraries return custom library and embedded library merged, overwriting embedded library functions, if necessary
+func mergeLibraries(customLib, embeddedLib string) (string, error) {
+	if customLib != "" {
```

Comment:
invert logic and eliminate `else`

```suggestion
    customLib = embeddedLib
	if customLib == "" {
	  // etc...
	}
	return customLib, nil
```